metadata-ws/src/test/resources/features/study.feature

```diff
@@ -688,7 +691,7 @@ Feature: study object
     Examples:
       | base          | query             | url    |
       | taxonomy-id   | id=9606           | STUDY2 |
-      | taxonomy-name | name=Homo sapiens | STUDY2 |
+      | taxonomy-name | name=Homo Sapiens | STUDY2 |
```

Comment:
It was actually correct the first time around, because species name is written starting with a lowercase letter
https://en.wikipedia.org/wiki/Binomial_nomenclature